spec/models/article_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  let!(:article) { FactoryBot.create :article }
+  let(:title1) { FactoryBot.build :article, title: nil }
+  let(:title2) { FactoryBot.build :article, title: "abcd" }
+  let(:title3) { FactoryBot.build :article, title: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." }
+  let(:text) { FactoryBot.build :article, text: nil }
+
+  context "article" do
+    it "created" do
```

Comment:
we can refactor it like this
```suggestion
    it { expect(article).to be_valid }
```

Reply:
or you can also do `it "is valid with valid attributes"`